modules/channel-messenger/README.md

```diff
@@ -0,0 +1,7 @@
+# Channel Messenger
+
+## How to setup
+
+1. Follow the instructions to setup your Facebook App: https://developers.facebook.com/docs/messenger-platform/getting-started/app-setup
+1. Make sure the module is built: `yarn cmd build:modules`
+1. Paste your Facebook App Token in the `verifyToken` property of your `data/bots/<your_bot>/config/channel-messenger.json`
```

Comment:
i'd move the thorough doc on how to set it up here

Reply:
So the more in-depth tutorial would be in the module's README and the doc would point to it? Yeah I like it.